src/braille/qml/MuseScore/Braille/BrailleView.qml

```diff
@@ -173,7 +173,7 @@ StyledFlickable {
 
         NavigationControl {
             id: fakeNavCtrl
-            name: "Braille"
+            name: "Braille" + brailleModel.brailleModeChanged()
```

Comment:
Oops ! Sry for that

Reply:
`brailleModel.mode ` gives 1 or 2. The `brailleModeChanged()` function sets an accessible string using a switch case based on this value. That's why I call `brailleModeChanged()`.

Reply:
But brailleModeChanged is a signal, and doesn't return something. So it doesn't make sense to assign it to a property. Also, it isn't supposed to be called from outside the model; it should only be emitted from within the model in C++. 

But it looks like emitting the signal is not the problem. The problem is that the name is set in the handler of the signal, instead of as a QML binding. 

So:
1. Remove the onBrailleModelChanged handler
2. For the navigation control, write: 
```c++
accessible.name: {
    switch (brailleModel.mode) {
        case 1: return "..."
        case 2: return "..."
    }
}
```

Also, actually, because this is a user-exposed string, it should be translatable. You do that by writing `qsTrc("braille", "<the English text>")`.